src/airport.js

```diff
@@ -1,5 +1,7 @@
 class Airport {
 
+  airportCapacity = 5;
```

Comment:
Here I am not using private properties in my Classes, if I were to do this again now I would ensure all of the properties were private by using # before the property name and also code getter and setter methods where appropriate.